### ISSUE TITLE
Global channel map doesn't work correctly in batch scripts

### ISSUE THREAD

**Issue by rbeckman-nextgen** [nextgenhealthcare/connect#3612]
All batch scripts are incorrectly using the "batch script ID" instead of the channel ID when getting a global channel map instance. Because of that, the map provided in the JavaScript context is actually different from the one used everywhere else by the channel.

Imported Issue. Original Details:
Jira Issue Key: MIRTH-3726
Reporter: narupley
Created: 2015-06-03T09:50:27.000-0700

**Comment by rbeckman-nextgen** [comment]
Fixed in revisions 7691/7692. The channel ID in the context should be correct now, so the "channelId" variable should be correct, and the global channel map should work as expected.

Imported Comment. Original Details:
Author: narupley
Created: 2015-06-03T09:58:50.000-0700

**Comment by rbeckman-nextgen** [comment]
Verified that the globalChannelMap within the batch script was different within the scripting context. I tested this by simply putting a variable into the globalChannelMap upon deploy, attempting to log this statement out in the server log. In 3.2.1, the variable was empty while in 3.2.2 the variable was successfully logged.

Imported Comment. Original Details:
Author: jaysenp
Created: 2015-06-03T17:23:50.000-0700
